CODING_STYLE.md

```diff
@@ -55,6 +55,7 @@ To set indentation and tab width settings uniformly, the repository contains an
 3. All symbols should be declared in a namespace except for final applications.
 4. Use anonymous namespaces for helpers whose scope is a cpp file only.
 5. Preprocessor symbols should be prefixed with the namespace in all-caps and an underscore.
+6. Do not use `std::` qualifier in cpp files (see 2.), except for `std::move`, which will otherwise cause the `check_style` step to fail.
```

Comment:
The rule is actually a bit more general. We just do not add any namespace qualifier when we already have `using namespace` in the file. The `std` is just the most prominent example because we have it everywhere and it's the easiest one to spot in reviews :)